app/Http/Controllers/SiteController.php

```diff
@@ -0,0 +1,258 @@
+<?php
```

Comment:
Controllers
 --- Subscriber/SubscriberController.php
 --- Auth/AuthController.php
 --- Contract/ContractController.php
.....

Models
 --- Subscriber/Subscriber.php
 --- Auth/Auth.php
 --- Contract/Contract.php
....

Same for service classes